parm/config/config.resources

```diff
@@ -193,6 +193,28 @@ elif [[ "${step}" = "atmanalpost" ]]; then
 
 elif [[ "${step}" = "aeroanlinit" ]]; then
 
+   # below lines are for creating JEDI YAML
+   case ${CASE} in
+     C768)
+        layout_x=6
+        layout_y=6
+        ;;
+      C384)
+        layout_x=5
+        layout_y=5
+        ;;
+     C192 | C96 | C48)
+        layout_x=8
+        layout_y=8
+        ;;
+      *)
+          echo "FATAL: Resolution not supported'"
```

Comment:
```suggestion
          echo "FATAL ERROR: Resolution not supported for aerosol analysis'"
```
NCO EE2 says use `FATAL ERROR`

Reply:
Probably also on L241.
At some point we will go through all of these, but every little bit helps.